docker/forge/Dockerfile

```diff
@@ -22,7 +22,12 @@ RUN IMAGE_TARGETS="test" ./docker/build-common.sh
 FROM debian:buster-20210511@sha256:acf7795dc91df17e10effee064bd229580a9c34213b4dba578d64768af5d8c51
 
 RUN apt-get update && apt-get install -y libssl1.1 openssh-client wget && apt-get clean && rm -r /var/lib/apt/lists/*
-RUN cd /usr/local/bin && wget "https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl" -O kubectl && chmod +x kubectl
+
+RUN mkdir /diem
+COPY rust-toolchain /diem/rust-toolchain
```

Comment:
dev_setup.sh is failing without this file. can probably remove this restriction for operational tools

Reply:
got around it for now

Reply:
you probably want -o for operations which installs helm.

Reply:
ah yea I misread the options. doing `-o -b`

Reply:
Failing to install shellcheck on the latest codebuild run https://us-west-2.console.aws.amazon.com/codesuite/codebuild/projects/diem-forge/build/diem-forge:46907ab4-9305-4546-ae68-eecab46dece0/. Any ideas?

```

Reading state information...
--
1405 | E: Unable to locate package xz
1406 | The command '/bin/sh -c /diem/scripts/dev_setup.sh -o -b' returned a non-zero code: 100
1407 | docker build failed. retrying in 10 secs.


```

Reply:
Eh added an option `-i` that lets us choose which exact packages to install. e.g. `-i helm -i kubectl`.